_spinnaker/policy_engine.md

```diff
@@ -45,7 +50,16 @@ armory:
     url: http://opa.opaserver:8181/v1
 ```
 
-After you update `front50-local.yml`, deploy your changes:
+
+If you'd like to enable individual features, add the above configuation to the following files:
```

Comment:
So adding the YAML to `front50` enables the pipeline configs, and clouddriver for runtime. Is there a single *-local.yml to add it to that'll turn on both (and all upcoming features?). Line 33 implies that. If not, I'm going to rework this section a little.

Reply:
Yep, `spinnaker-local.yml` would enable it for both.